src/patternfly/components/DataList/data-list.scss

```diff
@@ -85,24 +85,26 @@ $pf-data-list--breakpoint--mobile: 380px;
   }
 
   // Expandable content
-  --pf-c-data-list__expandable-content--Transition: var(--pf-global--Transition);
   --pf-c-data-list__expandable-content--BorderTopColor: var(--pf-global--Color--light-200);
-  --pf-c-data-list__expandable-content--PaddingRight: var(--pf-global--spacer--xl);
-  --pf-c-data-list__expandable-content--mobile--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-data-list__expandable-content--PaddingLeft: var(--pf-global--spacer--xl);
-  --pf-c-data-list__expandable-content--mobile--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-data-list__expandable-content--MarginRight: calc(var(--pf-c-data-list__expandable-content--PaddingRight) * -1);
-  --pf-c-data-list__expandable-content--MarginLeft: calc(var(--pf-c-data-list__expandable-content--PaddingLeft) * -1);
+  --pf-c-data-list__expandable-content--MarginRight: calc(var(--pf-c-data-list__expandable-content-body--PaddingRight) * -1);
+  --pf-c-data-list__expandable-content--MarginLeft: calc(var(--pf-c-data-list__expandable-content-body--PaddingLeft) * -1);
   --pf-c-data-list__expandable-content--BoxShadow: var(--pf-global--BoxShadow--md-bottom);
+  --pf-c-data-list__expandable-content--MaxHeight: #{pf-size-prem(600px)};
```

Comment:
do we want a `max-height` on this? Should it match the table, where you only get a max-height on mobile?

Reply:
Thats a great idea